HISTORY.rst

```diff
@@ -6,6 +6,30 @@ Release History
 Upcoming
 ++++++++
 
+New Release?
+++++++++++++++++++
+
+**Breaking Changes**
+
+- ``Events.get_events(...)`` now returns a list of ``Event`` instances rather than a list of ``dicts``
+  representing ``Event`` s.  ``Event`` inherits from ``Mapping`` but will not have all the same capabilities as
+  ``dict`` .
```

Comment:
Let's add a sub-bullet about how to migrate. Something like:

- .
  - Your code is affected if you use `Events.get_events(...)` and expect a list of `dict` instead of a list of `Mapping`. For example, if you use `__setitem__` (`event['key'] = value`), `update()`, or `copy()`, or if your code depends on the `str` or `repr` of the event. Use of `__getitem__` (`event['key']`), `get`, and other `Mapping` methods is unaffected. See https://docs.python.org/2.7/library/collections.html#collections-abstract-base-classes for methods supported on `Mapping` instances.
  - How to migrate: if you still have the need to treat the events like a `dict`, then you can get a deepcopy of the original `dict` with the new attribute `response_object`.